pkg/utils/networking.go

```diff
@@ -32,15 +33,21 @@ func getSubnetByInd(ind int) (*net.IPNet, error) {
 	return curNet, err
 }
 
-func getIPByInd(ind int) (net.IP, error) {
+func getIPByInd(ind int) ([]net.IP, error) {
 	if ind < 0 || ind > 255 {
 		return nil, fmt.Errorf("error in index %d", ind)
 	}
 	IP := net.ParseIP(fmt.Sprintf("192.168.%d.10", ind))
 	if IP == nil {
 		return nil, fmt.Errorf("error in ParseIP for index %d", ind)
 	}
-	return IP, nil
+	ips := []net.IP{IP}
+	IP2 := net.ParseIP(fmt.Sprintf("192.168.%d.11", ind))
```

Comment:
Magic number?

Reply:
No. It is an IP address. Needed to allocate another static IP address for a different ethernet interface on the emulated machine. Copied from what is there 4 lines above.

Reply:
got it.